frontend/scripts/selectors/years.selectors.js

```diff
@@ -0,0 +1,25 @@
+import { createSelector } from 'reselect';
+import intersection from 'lodash/intersection';
+
+export const makeGetAvailableYears = (
+  getSelectedYears,
```

Comment:
This is cool! I didn't know that you could pass the selectors as attributes of a factory